packages/@lwc/engine-core/src/framework/reporting.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { noop } from '@lwc/shared';
+import { VM } from './vm';
+
+export const enum ReportId {
+    CrossRootAriaInSyntheticShadow,
```

Comment:
TypeScript will auto-assign an incremented value for each entry in the enum. While this is an issue for now as we only have a single entry in the enum, this might be problematic as we start adding more entries. I would suggest adding an initial value for all the entries here to make it stable as we start adding/removing report ids.